libraries/Microsoft.Bot.Builder.Ai.LUIS/RecognizerResult.cs

```diff
@@ -1,12 +1,20 @@
-﻿using Newtonsoft.Json.Linq;
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Bot.Builder.Ai.LUIS
 {
     public class RecognizerResult
     {
+        [JsonProperty("text")]
```

Comment:
This is great. 

There should be a test that enforces these. 

Specifically, create an instance of this, serialize it to JSON, then compare it against a hand-coded and/or known good instance of this JSON doc.

Reply:
@cleemullins this test you described would just be testing the JSON.net library, doesn't seem very useful to me?

Reply:
Well, I guess it'd also be checking for any typos in the attribute names :)

Reply:
@cleemullins I added a unit test